MasterScript.py

```diff
@@ -92,9 +95,10 @@ def run_crispr_identify(args):
     """
 
     cur_path = str(pathlib.Path().absolute())
+    # WHY do you ignore the "--result_folder" argument?!?
     dirname_identify = cur_path + '/tmp/output-CRISPRidentify'
 
-    result = os.system('python3.7 CRISPRidentify/CRISPRidentify.py --file ' + args.fasta_file +
```

Comment:
subpath calls are not working, if you are running the script from another working directory (which we do..)